### ISSUE TITLE
Ditch "Priority Notifications" for Fully Fine-Tuning of Notification Experience

### ISSUE THREAD

**Issue by ThatOneUnoriginal** [bluesky-social/social-app#5097]
**Is your feature request related to a problem? Please describe.**

Relatively recently Bluesky started to experiment with notification filters and more specifically what it's calling priority notifications. This system makes it so you only receive replies and quote notifications from users you follow. The problem with this system currently is that the platform is deciding for you what is a priority and what isn't instead of the user deciding on their own what is and isn't.

**Describe the solution you'd like**

Instead of having a system where the platform decides what is a "priority" or not, the user should be able to disable and enable notifications for each individual action. For example, you could enable/disable notifications for; mentions, comments on posts, likes on your posts, likes on your comments, replies to your comments, etc... A good example of a system like this would be Reddit which allows you to modify the notifications for basically any and all actions that would result in a notification on a individual basis.

**Describe alternatives you've considered**

I personally love the system that Reddit uses for notifications, so I haven't really thought of any alternatives. It's the system that gives the end-user the most choice which I believe is something that should be encouraged.

**Additional context**

I understand that the notifications settings are still a work in progress which means that work on the feature is far from complete, and I'm sure some others have suggested something similar to this before (in the case of that, I apologize for the duplicate). But I believe that it would be better to push this idea whilst its in development/being worked on in case it hasn't been considered.
![Screenshot 2024-09-01 234749](https://github.com/user-attachments/assets/bbda43b1-5247-4633-aa88-119ec9325e7a)

**Comment by mistydemeo** [comment]
I agree this would be ideal. Twitter, Instagram, Ivory, etc. all have similar features. The Bluesky app is currently the only social media app I use that doesn't allow configuring which categories of notifications to send as push notifications.

**Comment by cooljeanius** [comment]
Yeah, this is getting more important with the recent influx of users; I've kinda pulled back on actually using the app lately due to drowning in so many notifications

**Comment by zeusshomaly** [comment]
@ThatOneUnoriginal I agree. I want the ability to receive notifcations even when i'm outside of bsky.app. Brave has notifications, but Bluesky doesn't support browser notifications just yet.

**Comment by cooljeanius** [comment]
I believe this can be closed as fixed now as of the latest release

**Comment by zeusshomaly** [comment]
> I believe this can be closed as fixed now as of the latest release

<img width="1048" height="801" alt="Image" src="https://github.com/user-attachments/assets/f3b4aa43-9913-48e9-85da-28354a6a8d01" />You're right! wow! Thanks!

**Comment by zeusshomaly** [comment]
@cooljeanius you are a legend, thank you so much for telling me, this really did just make my day! Kind regards, from Australia :)
